invokeai/frontend/web/src/services/api/endpoints/images.ts

```diff
@@ -208,6 +209,15 @@ export const imagesApi = api.injectEndpoints({
         try {
           const { data } = await queryFulfilled;
 
+          if (data.deleted_images.length < imageDTOs.length) {
+            dispatch(
+              addToast({
+                title: 'One or more images could not be deleted',
+                status: 'warning',
+              })
+            );
+          }
```

Comment:
@psychedelicious this ok to have in OSS? handles authorization failures (or possibly any back-end error with deleting bulk images)

Reply:
Yep I think this is something we should have anyways 👍